docs/resie_energy_system_components.md

```diff
@@ -345,198 +347,223 @@ Symbol | Description | Unit
-\(\eta_{HEL,LE}\) | efficiency of the power electronics of the electrolyser | [-]
-\(PL_{HEL,min}\) | minimum allowed partial load of the electrolyser | [-]
-\(MOT_{HEL}\) | minimum operating time of the electrolyser | [min]
-\(SUT_{HEL}\) | start-up time of the electrolyser until full heat supply (linear curve) | [min]
-\(CDT_{HEL}\) | cool-down time of the electrolyser from full heat supply to ambient (linear curve) | [min]
-\(e_{H_2} \) | mass-dependent energy of hydrogen (net calorific value or gross calorific value)
+\(P_{el,rated}\) | total electric power consumption of the electrolyser under full load (operating state 100 %) | [W]
+\(N_{unit}\) | number of units that make up the electrolyser plant | [-]
+dispatch strategy | method of dispatching the units of the electrolyser to meet demand | [-]
+\(\eta_{H_2}(\kappa)\) | efficiency of hydrogen production of each unit as function of \(\kappa\) | [-]
+\(\eta_{H_2,loss}(\kappa)\) | percentage of hydrogen losses of each unit as function of \(\kappa\) | [-]
+\(\eta_{heat,high}(\kappa)\) | efficiency of high temperature heat production of each unit as function of \(\kappa\) | [-]
+\(\eta_{heat,low}(\kappa)\) | efficiency of low temperature heat production of each unit as function of \(\kappa\) | [-]
+\(\kappa_{min,unit}\) | minimum \(\kappa\) of each unit | [-]
+\(\kappa_{min}\) | minimum total \(\kappa\) of the electrolyser | [-]
+\(\kappa_{opt}\) | optimal \(\kappa\) of each unit at which \(H_2\) production is most efficient | [-]
+\(e_{H_2} \) | mass-dependent energy of hydrogen (net calorific value or gross calorific value) | [Wh/kg]
 \(v_{O_2,H_2} \) | stoichiometric mass-based ratio of oxygen and hydrogen supply during electrolysis | [kg \(O_2\) / kg \(H_2\)]
-\(\eta_{H_2 \ purification} \) | percentage of purification losses in hydrogen purification | [%]
-\(\eta_{H_2O \ treatment} \) | percentage of purification losses in water treatment | [%]
-\(p_{H_2,HEL} \) | pressure of hydrogen supply | [bar]
-\(p_{O_2,HEL} \) | pressure of oxygen supply | [bar]
-\(T_{HEL,cooling,in,max}\) | max. allowed temperature of cooling medium input | [°C]
+\(T_{heat,high}\) | cooling HT fluid outlet temperature of electrolyser | [°C]
+\(T_{heat,low}\) | cooling LT fluid outlet temperature of electrolyser | [°C]
 
+### Typical efficiency functions
+**Note:** These are exemplary values and do not imply validation or extensive research.
 
-**State variables of the Electrolyser:**
+**Constant efficiencies**
+
+* Adapted from a realised \(1 \ MW_e\) PEM electrolyser described in Fisch2023[^Fisch2023] from the data of 2023, a year of mostly nominal operation. Of note is that the electrolyser was still in the process of being improved. The values are thus perhaps somewhat pessimistic for the expected efficiency.
```

Comment:
Are we allowed to publish the numbers? Should we cite the latest poster from EWB Meeting in Kassel?